### PR TITLE
fix psp for startupapicheck from upstream

### PR DESCRIPTION
**Pull request by ubergesundheit**
Towards giantswarm/giantswarm#19074

This PR:

- fixes installation issues

#### Optional app

- [ ] fresh install
- [ ] upgrade from previous version

#### Pre-installed app

- [x] fresh install during cluster creation
 - tested on `gauss` with `16.0.0-calvix` and `15.3.0-njuettner`
- [ ] upgrade from previous version in a pre-existing cluster

#### Other testing

- [ ] check reconciliation of existing resources after upgrading

### Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: helm/cert-manager-app/templates/startupapicheck-psp-clusterrole.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{{- if .Values.startupapicheck.enabled -}}
+kind: ClusterRole
```
Comment (webwurst) [inline helm/cert-manager-app/templates/startupapicheck-psp-clusterrole.yaml:2]: Any reason to not picking up `{{- if .Values.global.podSecurityPolicy.enabled }}` from [upstream](https://github.com/jetstack/cert-manager/blob/master/deploy/charts/cert-manager/templates/startupapicheck-psp-clusterrole.yaml)? Just would be nice to follow upstream as close as possible.

**Comment by webwurst** [comment]
Maybe I am overlooking something, but shouldn't we use an image from quay.io/giantswarm [here](https://github.com/giantswarm/cert-manager-app/blob/e41c1c78fdb3503504475fd0912ebdd705e08073/helm/cert-manager-app/values.yaml#L185)? And why we are at it we could maybe switch to `1.5.4`.

**Comment by ubergesundheit** [comment]
> Maybe I am overlooking something, but shouldn't we use an image from quay.io/giantswarm [here](https://github.com/giantswarm/cert-manager-app/blob/e41c1c78fdb3503504475fd0912ebdd705e08073/helm/cert-manager-app/values.yaml#L185)? And why we are at it we could maybe switch to `1.5.4`.

:sweat_smile: totally forgot

**Review by webwurst — APPROVED** [review]

File: helm/cert-manager-app/templates/startupapicheck-psp-clusterrole.yaml (hunk 1 above)
Comment (ubergesundheit) [reply helm/cert-manager-app/templates/startupapicheck-psp-clusterrole.yaml:2]: While following upstream as close as possible is a very good thing, PSP is required in GS clusters anyway, so trying to keep configuration as simple as possible

Comment (webwurst) [reply helm/cert-manager-app/templates/startupapicheck-psp-clusterrole.yaml:2]: oki

**Review by njuettner — APPROVED** [review]
lgtm

**Pull request merged**